server/test/search.spec.js

```diff
@@ -0,0 +1,75 @@
+import chai, { expect } from 'chai';
+import chaiHttp from 'chai-http';
+
+import app from '../..';
+
+chai.use(chaiHttp);
+
+describe('Search Articles Endpoint /articles?query=params', () => {
```

Comment:
Good Job! But kindly go through the comments I made.

Reply:
Thanks